src/analysis/analytic/composition.lean

```diff
@@ -164,6 +165,12 @@ by simp [apply_composition]
 
 end formal_multilinear_series
 
+variables [nondiscrete_normed_field 𝕜]
```

Comment:
I didn't attempt to generalize further down the file, although it can certainly be done.